### PR TITLE
TF-01-010-Tutoring

### LINKED ISSUE #40: TF-01-010-Tutoring
teachworks com_tutoring-management-software

### PR DESCRIPTION
**Pull request by jobayeructit**
[screen-capture (2).webm](https://github.com/UCT-IT/edume/assets/149924386/2f2716eb-c476-4d6e-89c1-2c5d5a2fb46c)
screencapture-localhost-3000-pages-solution-tutoring-2023-11-28-17_04_17

### CODE REVIEW

File: src/components/common/commonCarousel/CommonCarousel.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
+"use client";
+import { motion } from "framer-motion";
+import Slider from "react-slick";
+import "slick-carousel/slick/slick-theme.css";
+import "slick-carousel/slick/slick.css";
+import Link from "next/link";
+import NextArrow from "../carouselArrows/NextArrow";
+import PrevArrow from "../carouselArrows/PrevArrow";
+
+type carousel = {
```
Comment (saminsss) [inline src/components/common/commonCarousel/CommonCarousel.tsx]: Uppercase type name

File: src/components/common/commonCarousel/CommonCarousel.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
+"use client";
```
Comment (saminsss) [inline src/components/common/commonCarousel/CommonCarousel.tsx:1]: This is necessary?

File: src/app/pages/solution/tutoring/page.tsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+              hover="bg-[#97CE2C]"
+            />
+          </div>
+        </div>
+      </HeroBackground>
+      <TutorScheduling />
+      <CommonSection
+        leftComponent={<SaveTimeLeftComponent />}
+        rightComponent={<SaveTimeRightComponent />}
+        orderFirst="first"
```
Comment (saminsss) [inline src/app/pages/solution/tutoring/page.tsx]: Change this to boolean

**Review by saminsss — CHANGES_REQUESTED** [review]

File: src/components/common/commonCarousel/CommonCarousel.tsx (hunk 2 above)
Comment (jobayeructit) [reply src/components/common/commonCarousel/CommonCarousel.tsx:1]: If I don't use "use client" I get this error
Error: (0 , react__WEBPACK_IMPORTED_MODULE_0__.createContext) is not a function

File: src/components/common/commonCarousel/CommonCarousel.tsx (hunk 1 above)
Comment (jobayeructit) [reply src/components/common/commonCarousel/CommonCarousel.tsx]: done

File: src/app/pages/solution/tutoring/page.tsx (hunk 3 above)
Comment (jobayeructit) [reply src/app/pages/solution/tutoring/page.tsx]: changed it to boolean

**Review by saminsss — APPROVED** [review]

**Pull request merged**